administrator/language/en-GB/plg_finder_contacts.ini

```
@@ -5,6 +5,12 @@
 
 PLG_FINDER_CONTACTS="Smart Search - Contacts"
 PLG_FINDER_CONTACTS_XML_DESCRIPTION="This plugin indexes Joomla! Contacts."
+PLG_FINDER_CONTACTS_TAXONOMIES_CATEGORY="Category"
+PLG_FINDER_CONTACTS_TAXONOMIES_COUNTRY="Country"
+PLG_FINDER_CONTENTS_TAXONOMIES_LABEL="Taxonomies to index"
```

```suggestion
PLG_FINDER_CONTENTS_TAXONOMIES_LABEL="Taxonomies to Index"
```
Please repeat with the other instances.